### DISCUSSION TITLE
Type check and moduler system

### DISCUSSION THREAD

**Discussion by BhJaipal** [carbon-language/carbon-lang#4334] [Language design]
Is it possible to add a type comparison like python `type(var) == someClass` or `type(var) == int`to carbon
And like rust and nodejs, it should export functions, variables and classes we like to export and not everything

**Comment by jonmeow** [comment]
We'd like to have something like that, but our focus right now is the [0.1 milestone](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/milestones.md). There are just a lot of features which we don't have enough people to design and implement in the short-term. If you'd like to contribute to the progress of Carbon, our [contributing guide](https://github.com/carbon-language/carbon-lang/blob/trunk/CONTRIBUTING.md) gives some tips, and there's #contributing-help on [Discord](https://discord.gg/ZjVdShJDAs).

**Reply by BhJaipal** [reply]
> We'd like to have something like that, but our focus right now is the [0.1 milestone](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/milestones.md). There are just a lot of features which we don't have enough people to design and implement in the short-term. If you'd like to contribute to the progress of Carbon, our [contributing guide](https://github.com/carbon-language/carbon-lang/blob/trunk/CONTRIBUTING.md) gives some tips, and there's #contributing-help on [Discord](https://discord.gg/ZjVdShJDAs).

Ok, do I have to learn LLVM? Where can I learn it from, I do have used C++ so I would like to try to help
